superset/databases/api.py

```diff
@@ -311,15 +239,11 @@ def select_star(
             description: Table schema
           responses:
             200:
-              description: select star for table
+              description: Query result
```

Comment:
Does this actually return a query result? I thought it returned a SQL string representing the verbose equivalent of `SELECT * FOR <table_name>` for the referenced table.

Reply:
Totally right, nice catch